test/middleware.test.js

```diff
@@ -126,10 +126,30 @@ describe(`middleware: ${printPath("[test/middleware.test.js]")}`, function () {
             res.status(200).json({ message: req.session.getUserId() });
         });
 
-        app.get("/user/handle", Session.verifySession(true), async (req, res) => {
+        app.get("/user/handleV0", Session.verifySession(true), async (req, res) => {
```

Comment:
There are way too many test changes, and they seem to follow a similar pattern. You will need to guide me through them.